katdal/chunkstore_s3.py

```diff
@@ -168,6 +168,15 @@ def _read_chunk(response):
     return chunk
 
 
+def _bucket_url(url):
```

Comment:
between the various repos we are repeating a lot of these kind of functions. It might be nice to pull these into a central repo at some point.

Reply:
As long as there is no danger of falling into the trap of https://github.com/ska-sa/katmisc, which has turned me off common utility libraries for a while. :-D

Reply:
haha, yeah, anything with the name misc is bound to cause confusion. Something along the lines of S3 utilities is probably more specific and is the kind of thing that you don't want to re-develop in each library...

Reply:
I count about 100-140 lines in katdal that qualifies as "S3 utility" (tests excluded). It will have to get more substantial to justify an entire package but it's good to keep an eye on it.